src/applications/gqrx/mainwindow.cpp

```diff
@@ -673,7 +673,7 @@ bool MainWindow::loadConfig(const QString& cfgfile, bool check_crash,
         }
 
         ui->freqCtrl->setFrequency(int64_val);
-        setNewFrequency(ui->freqCtrl->getFrequency()); // ensure all GUI and RF is updated
+        //setNewFrequency(ui->freqCtrl->getFrequency()); // ensure all GUI and RF is updated
```

Comment:
Please don't comment out code; just delete it. Revision control already keeps track of changes.

Reply:
Sorry, was unsure how you handle this. :confused: